roles/ovb/bmc/tasks/os-net-config.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Register management network details
+  os_networks_facts:
+      cloud: "{{ provisioner.cloud | default(omit) }}" 
+      filters:
+          name: "{{ provisioner.prefix | default(omit) }}-management"
```

Comment:
- Please add "prefix" to role defaults like you did for the rest of the roles.
- This is executed from bmc node. need to use localhost. Add to task:
```yaml
  os_networks_facts:
      args: {}
  delegate_to: localhost
  become: no
  register: management_network_details


```